craft_archives/repo/apt_preferences_manager.py

```diff
@@ -0,0 +1,140 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2015-2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
```

Comment:
I believe `craft-archives` is LGPL (the applications themselves are GPL)

Reply:
correct, considering that Charmcraft is Apache Licensed we went with LGPL for all our libraries.

Reply:
Yeah looks like the LICENSE file is correct, but the file headers aren't. I'll do another PR to fix this on the other files too.